### DISCUSSION TITLE
Why no `pkg` completions?

### DISCUSSION THREAD

**Discussion by Rudxain** [termux/termux-packages#16568] [Ideas]
I think `pkg` should have shell completions, even though most subcommands have "shortcut" alternatives

**Comment by TomJo2000** [comment]
That would be dependent on the shell's completion facilities and would most likely require separate completions for each shell.
It's a cool idea, it'd also take a bunch of work.

And completing package names for installation is pretty much out of the question unless you wanna dynamically scrape them from the local `apt`/`pacman` repo content lists.

I might write a proof of concept completion for it for Bash or Zsh if I find the time.

**Comment by sylirre** [comment]
If someone want such extra functionality as shell completions, please submit PRs to this repo: https://github.com/termux/termux-tools
